relay-server/src/services/spooler/sql.rs

```diff
@@ -74,7 +74,9 @@ pub fn delete<'a>(key: QueueKey) -> Query<'a, Sqlite, SqliteArguments<'a>> {
 ///
 /// This info used to calculate the current allocated database size.
 pub fn current_size<'a>() -> Query<'a, Sqlite, SqliteArguments<'a>> {
```

Comment:
Can you update the docs for this function and the calling function?

Reply:
👍 fun fact: The doc on the calling function was already correct because we never updated it.

Reply:
Not quite, it didn't subtract the free pages.